h2o-docs/src/product/data-science/glm.rst

```diff
@@ -103,6 +103,8 @@ Defining a GLM Model
 
 -  `missing_values_handling <algo-params/missing_values_handling.html>`__: Specify how to handle missing values (Skip or MeanImputation).
```

Comment:
Please see https://github.com/h2oai/h2o-3/pull/3821/files#diff-1bc6d9a43cd27577320aabef1d6ded38L117

"...Specify how to handle missing values (Skip, MeanImputation, or PlugValues)."